### DISCUSSION TITLE
Ability to run linters only on files modified within the current PR

### DISCUSSION THREAD

**Discussion by cbugot-swi** [super-linter/super-linter#1617] [Ideas]
When wanting to add linter to existing/big project it might be useful to only lint the currently modified files. This gives achievable goals to developers to fix a few files at a time and get a green check in their PRs. Any chance this option could be added

**Comment by cbugot-swi** [comment]
Damned there is already an option for this, for some reason I missed it the first time I tried that action :)
For the ones landing here here is the option: VALIDATE_ALL_CODEBASE

**Reply by cbugot-swi** [reply]
@admiralAwkbar Can you close/mark as resolved that 'Idea' ?

**Reply by bewuethr** [reply]
I find that the option doesn't work for me, and pushes to PRs still trigger linting the entire codebase and not just changed files. Does it work for you?

**Reply by BretFisher** [reply]
I use it across dozens of projects and the defaults work fine in PRs against only changed files.

**Reply by bewuethr** [reply]
I opened an issue (#2548) to show how it doesn't work for me, but weirdly it now works about everywhere else! I wonder if `fetch-depth: 0` is the culprit...
